src/messages/SharedMessageBuilder.hpp

```diff
@@ -32,6 +33,11 @@ class SharedMessageBuilder : public MessageBuilder
     virtual void triggerHighlights();
     virtual MessagePtr build() = 0;
 
+    static std::pair<QString, QString> slashKeyValue(const QString &str);
```

Comment:
Mismatching definition
```suggestion
    static std::pair<QString, QString> slashKeyValue(const QString &keyValueString);
```

will most likely need a reformat after the suggestion too